pyproject.toml

```diff
@@ -34,7 +34,7 @@ pythonpath = [
   "."
 ]
 log_cli = true
-log_cli_level = "INFO"
+log_cli_level = "DEBUG"
```

Comment:
Can we pass this manually while debugging instead of making it the default? It's too much output.

Reply:
Ok, I like it chatty but happy to make that change.

Reply:
I think something like `pytest -v --log-cli-level=DEBUG tests` could be used locally, but it doesn't have to be the default.

Reply:
Yeah that command works perfectly